README.md

```diff
@@ -818,7 +818,7 @@ defaults write com.apple.finder AppleShowAllFiles true
 defaults write com.apple.finder AppleShowAllFiles false
 ```
 
-#### Remove Protected Flag
+#### Remove Locked Status
```

Comment:
This should be a comment rather than a change. In Unix, this is called a flag and the command is even called `chflags`. The fact that macOS uses a different terminology in Finder does not change that.

I suggest adding a line directly below here:

> This is equivalent to Finder "Locked" status.